public/index.html

```diff
@@ -101,6 +101,5 @@
 
 <div id="content"></div>
 <script src="https://code.jquery.com/jquery-3.1.0.js"></script>
-<script type="text/jsx" src="index.js"></script>
```

Comment:
Do we know which part of the build machinery re-inserts this for us? Is it the npm buld, or gh-pages? @eknuth